### ISSUE TITLE
Duplicate learn more text in notification popups

### ISSUE THREAD

**Issue by fofajardo** [FranklinDM/Moonscape#7]
It is still unknown if this is a browser-related bug or with the theme itself. 
Reported [here](https://forum.palemoon.org/viewtopic.php?f=45&t=14492&p=112297#p112290).

**Comment by fofajardo** [comment]
It's not a browser bug and is just a matter of a few missing lines:
```
.popup-notification-learnmore-link:not([href]) {
  display: none;
}
```

Based on the selector, this line kind of hides the learn more link when there's no assigned link, hence :not([href].

Fixed with commit fb0cefa1e2d7c5b3266fc50362b6162eecf12fb1

**Comment by fofajardo** [comment]
Tag #1
